src/mmw/js/src/draw/views.js

```diff
@@ -213,9 +213,10 @@ var PlaceMarkerView = Marionette.ItemView.extend({
         'change:toolsEnabled': 'render'
     },
 
-    onItemClicked: function() {
+    onItemClicked: function(e) {
         var self = this,
             map = App.getLeafletMap(),
+            itemName = e.target.innerHTML,
```

Comment:
Good catch. We expect `itemName` to be a text value, not an HTML value, but since in this case the `innerHTML` is just a plain text value we are getting away with it. `innerText` is not a browser standard. The alternative would be to use jQuery using `$(e.target).text()` which may be a little less efficient because it causes a DOM lookup but the clarity may well be worth it.